libs/expo/betterangels/src/lib/screens/AddEditClient/index.tsx

```diff
@@ -189,8 +229,34 @@ export default function AddEditClient({ id }: { id?: string }) {
 
     const { displayCaseManager, ...updatedClientInput } = data.clientProfile;
 
+    const existingSocialMediaProfiles =
+      data.clientProfile.socialMediaProfiles || [];
+
+    const updatedSocialMediaProfiles = defaultSocialMedias.map(
+      (defaultProfile) => {
+        const existingProfile = existingSocialMediaProfiles.find(
+          (profile) => profile.platform === defaultProfile.platform
+        );
+
+        if (existingProfile) {
+          const { __typename, ...cleanedProfile } = existingProfile;
```

Comment:
did you mean `cleanedProfile` or `clientProfile`?
if `cleanedProfile`, can you explain what that means?

Reply:
removing __typename